packages/grid/_modules_/grid/hooks/features/pagination/gridPaginationInterfaces.ts

```diff
@@ -0,0 +1,33 @@
+export interface GridPaginationState {
```

Comment:
I'm not sure how we should name this file
But I like the idea of having a single file with all the interfaces of a given feature hook.